src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+
+                    if ( !runningServers.isEmpty() ) {
+
+                        var stopMessage = "Stopping server #serverInfo.name# first....";
+        
+                        print.line( stopMessage )
+                            .toConsole();
+        
+                        runningServers.each( function( ID ){
+                            var stopResults = serverService.stop( runningServers[ arguments.ID ] );
+                            print.line( stopResults.messages, ( stopResults.error ? 'red' : 'green' ) )
+                                .toConsole();
+                        } );
+        
+                        // Give them a second or three to die or file locks will still be in place (on Windows, at least)
+                        // This is hacky, but there's no clean way to poll for when the process is 100% dead
+                        sleep( 3000 );
+                    }
+
+                    for( currentServer in filterServers ){
+
+                        print.line( serverService.forget( #currentServer.name# ), 'red' )
+                            .toConsole();
+                    }
+
+                } else {
+                    
+                    for( currentServer in filterServers ){
+
+                        var askMessage = "Really forget & delete server '#currentServer.name#' forever [y/n]?";
+            
+                        if( confirm( askMessage ) ){
+                            print.line( serverService.forget( #currentServer.name# ), 'red' )
+                                .toConsole();
+                        } else {
+                            print.line( "Cancelling forget '#currentServer.name#' command", 'blue' )
+                                .toConsole();
+                        }        
+                
+                    }     
+
+                
+                }
+
+            }
+
+        }
+
+
+    }
+
+    private function generatePruneListServers( required string months, required struct servers ){
+
+        /* user wants to see the list of servers with given months */
+        print.line( "generating list servers to be pruned...", 'yellow' ).toConsole();
+        serverCount=0;
+        data = [];
+        for( currentServer in servers )
+        {
+            lastStarted = servers[ currentServer ].dateLastStarted;
+            result = dateDiff( "m", servers[ currentServer ].dateLastStarted, now() );
+
+            serverData = [];
+            if ( result >= months ){
+                ArrayAppend( serverData, "#servers[ currentServer ].name#" );
+                ArrayAppend( serverData, "#servers[ currentServer ].dateLastStarted#" );
+                ArrayAppend( serverData, "#result#" );  
+                ArrayAppend( data, serverData );
+                serverCount += 1;
+            }
+
+        }
+
+        print.table( data=data, headerNames=[ "name", "last started", "months" ] )
```

Comment:
`print` calls can be chained like so for cleaner code:

```js
print.line()
  .line()
  .blueLine()
  .fooBarLine();
```

Reply:
![image](https://user-images.githubusercontent.com/80276017/124828929-290e5d00-df35-11eb-8414-b5e1023c6f00.png)